behat/features/tabs.feature

```diff
@@ -0,0 +1,144 @@
+Feature: Book page
+  As an anonymous user
+  We need to be able to see the tabs list in the book page.
+
+  @api
+  Scenario Outline: Visit a books page
```

Comment:
@miri this shouldn't be a ``Scenario Outline`` -- no need to reload the URL over and over again. Sec, I'll attach an example of what to use